src/estimation/include/iDynTree/Estimation/ExtendedKalmanFilter.h

```diff
@@ -254,7 +271,14 @@ namespace iDynTree
          */
         bool ekfGetStateCovariance(const iDynTree::Span<double> &P) const;
 
-    private:
+        /**
+        * function template to ignore unused parameters
+        */
+        template <typename T>
+        void ignore(T &&)
+        { }
```

Comment:
Can't this be private?

Reply:
If I put this private, it cannot be used by the inheriting classes.

Reply:
I will mark it protected